agent/src/agent/pipeline/json_builder/__init__.py

```diff
@@ -80,6 +80,7 @@ def _get_raw_builder(pipeline_: Pipeline, config: dict, is_edit: bool) -> IBuild
         source.TYPE_MYSQL: JDBCRawBuilder,
         source.TYPE_POSTGRES: JDBCRawBuilder,
         source.TYPE_SNMP: SNMPRawBuilder,
+        source.TYPE_KAFKA: KafkaBuilder,
```

Comment:
Isn't that the `KafkaRawBuilder` needed to be here? If not, then `KafkaRawBuilder` seems not used